config/initializers/fog.rb

```diff
@@ -0,0 +1,10 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider               => 'AWS',                        # required
+    :aws_access_key_id      => ENV['AWS_ACCESS_KEY_ID'],                        # required
+    :aws_secret_access_key  => ENV['AWS_SECRET_ACCESS_KEY'],                        # required
+  }
+  config.fog_directory  = ENV['S3_BUCKET']                          # required
+  config.fog_public     = false                                        # optional, defaults to true
+  config.fog_attributes = {'Cache-Control'=>"max-age=#{365.day.to_i}"} # optional, defaults to {}
```

Comment:
Line is too long. [97/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.